### PR TITLE
Improve Debian compliance on AT >= 13.0

### LINKED ISSUE #1563: Missing mandatory fields from deb control files
I noticed the field `Standards-Version` is missing in the first paragraph of the AT 14.0 deb control file:
https://github.com/advancetoolchain/advance-toolchain/blob/7c94f8029a47af021348ed0a977d5e71f5ced3e1/configs/14.0/deb/monolithic/control#L14-L20

But looks like it is mandatory ([link](https://www.debian.org/doc/debian-policy/ch-controlfields.html#source-package-control-files-debian-control)). 

That is true for other files as well.:
```
[advance-toolchain]$ for f in $(find -name 'control'); do grep -vl Standards-Version $f; done 
./configs/10.0/deb/monolithic/control
./configs/8.0/deb/monolithic/control
./configs/8.0/deb/monolithic_cross/control
./configs/13.0/deb/monolithic/control
./configs/13.0/deb/monolithic_cross/control
./configs/12.0/deb/monolithic/control
./configs/12.0/deb/monolithic_cross/control
./configs/7.1/deb/monolithic/control
./configs/7.1/deb/monolithic_cross/control
./configs/14.0/deb/monolithic/control
./configs/14.0/deb/monolithic_cross/control
./configs/9.0/deb/monolithic/control
./configs/9.0/deb/monolithic_cross/control
./configs/next/deb/monolithic/control
./configs/next/deb/monolithic_cross/control
./configs/11.0/deb/monolithic/control
./configs/11.0/deb/monolithic_cross/control
```

On these same files, the fields `Version` and `Maintainer` for each binary package are also missing, and are also mandatory as per the document linked above.

### PR DESCRIPTION
**Pull request by er-1**
Updated the format of the version to match with Debian's policy[1].
Fixed the maintainer value among configuration files.
Raised the compat level from 9 to 12.

Fix #1563

### CODE REVIEW

File: scripts/utilities/pkg_build_deb.sh [hunk 1]
```diff
@@ -86,6 +87,10 @@ for f in *; do
 	    -e "s/__USE_SYSTEMD__/${use_systemd}/g" \
 	    ${f} > ${deb_d}/${f}
 done
+if [[ -s ${deb_d}/changelog ]]; then
+	grep -v "^#" ${deb_d}/changelog > ${deb_d}/changelog.tmp
+	mv ${deb_d}/changelog.tmp ${deb_d}/changelog
```
Comment (ThinkOpenly) [inline scripts/utilities/pkg_build_deb.sh:92]: You could replace these two lines with:
```
  sed -i '/^#/d' ${deb_d}/changelog
```

File: configs/13.0/deb/monolithic/compat [hunk 2]
```diff
@@ -1 +1 @@
-9
+12
```
[the remark this answers is not in the document]
Comment (er-1) [reply configs/13.0/deb/monolithic/compat:1]: I think it's easier to follow compat updates (including fixes + improvements) from time to time than making a big step from a deprecated level to the latest for example. That said, I think we don't have to follow each compat update.

File: scripts/utilities/pkg_build_deb.sh [hunk 3]
```diff
@@ -70,7 +71,7 @@ go_dest="/usr/local/go"
 for f in *; do
 	echo "Setting up ${deb_d}/${f}"
 	sed -e "s/__AT_DEST__/${at_dest//\//\\/}/g" \
-	    -e "s/__AT_FULL_VER__/${at_full_ver}/g" \
+	    -e "s/__AT_FULL_VER__/${at_full_ver_deb_format}/g" \
```
[the remark this answers is not in the document]
Comment (er-1) [reply scripts/utilities/pkg_build_deb.sh:74]: This is why this PR is a draft. I haven't checked that yet.

File: configs/13.0/deb/monolithic/rules [hunk 4]
```diff
@@ -33,15 +33,12 @@ binary-arch: build binary-cp
...
-ifeq (__USE_SYSTEMD__, yes)
-	dh_systemd_enable
-endif
 
 # Copy the packages' files.
 	dh_install
 	dh_missing --fail-missing
 ifeq (__USE_SYSTEMD__, yes)
-	dh_systemd_start
+	dh_installsystemd
```
[the remark this answers is not in the document]
Comment (er-1) [reply configs/13.0/deb/monolithic/rules:41]: Right, this change comes with the update of `compat`. In `compat` 12, `dh_systemd_start` is a default behavior of `dh_installsystemd`.

Why "level 11, actually" ?

File: configs/13.0/deb/monolithic/control [hunk 5]
```diff
@@ -15,7 +15,7 @@
 Source: advance-toolchain-__AT_MAJOR_INTERNAL__
 Section: devel
 Priority: optional
-Maintainer: Advance Toolchain
+Maintainer: Advance Toolchain <at@build>
```
[the remark this answers is not in the document]
Comment (tuliom) [reply configs/13.0/deb/monolithic/control:18]: AFAIK, we do not have.
Does Debian accept an URL here?

[the remark this answers is not in the document]
Comment (er-1) [reply configs/13.0/deb/monolithic/control:18]: As said [here](https://www.debian.org/doc/debian-policy/ch-controlfields.html#s-f-maintainer), it's a name, and then an email address inside angle brackets. So we can't put an URL.

File: scripts/utilities/pkg_build_deb.sh (hunk 3 above)
[the remark this answers is not in the document]
Comment (er-1) [reply scripts/utilities/pkg_build_deb.sh:72]: I checked, it works.
For the purpose of the test, I locally released a cross build of AT 13.0-6 on Ubuntu 18.04. Then I updated a machine with a previous installed AT 13.0-5 (taken from the [official repository](https://public.dhe.ibm.com/software/server/POWER/Linux/toolchain/at/)):
```
$ sudo apt-get upgrade 
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Calculating upgrade... Done
The following packages will be upgraded:
  advance-toolchain-at13.0-cross-common advance-toolchain-at13.0-cross-ppc64le
2 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
Need to get 0 B/374 MB of archives.
After this operation, 364 MB disk space will be freed.
Do you want to continue? [Y/n] 
Get:1 ████████████████████████████████████████████ bionic/at13.0 amd64 advance-toolchain-at13.0-cross-ppc64le amd64 13.0.6 [368 MB]
Get:2 ████████████████████████████████████████████ amd64 advance-toolchain-at13.0-cross-common amd64 13.0.6 [5,462 kB]
(Reading database ... 126318 files and directories currently installed.)
Preparing to unpack .../advance-toolchain-at13.0-cross-ppc64le_13.0.6_amd64.deb ...
Unpacking advance-toolchain-at13.0-cross-ppc64le (13.0.6) over (13.0-5) ...
Preparing to unpack .../advance-toolchain-at13.0-cross-common_13.0.6_amd64.deb ...
Unpacking advance-toolchain-at13.0-cross-common (13.0.6) over (13.0-5) ...
Setting up advance-toolchain-at13.0-cross-common (13.0.6) ...
Setting up advance-toolchain-at13.0-cross-ppc64le (13.0.6) ...
Processing triggers for libc-bin (2.27-3ubuntu1.4) ...
```

File: scripts/utilities/pkg_build_deb.sh (hunk 1 above)
Comment (er-1) [reply scripts/utilities/pkg_build_deb.sh]: Done.

**Comment by er-1** [comment]
Closing this because:
 - part of it will be included in the support of Debian 11 (rise of compat level)
 - not sure about the change of version, this could add confusion between deb and rpm packages

**Pull request closed without merging**